### PR TITLE
InnerContent, Modernize Underlying Libraries and Update Dependencies

### PR DESCRIPTION
**Pull request by kuasha420**
Hello,

Great work with the library!

I was trying to add a custom component which required the innerContent to be passed to the block. So I had to make some changes to the library. 

While i was there, I also made react, react-dom and @loadable/component Peer Dependency,  Moved all dependencies to Dev Dependency and updated them to the latest version. This will ensure lesser package size and less chance of conflicts. 

I have also changed all instances of SFC to FC as the latter is depreciated. 

Please have a look and let me know if anything needs changing. 

Cheers!

### CODE REVIEW

File: dist/blocks/common/audio.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ const WPGAudioBlock = (props) => {
     const { 
     // attrs,
     // innerBlocks,
-    innerHTML } = props;
+    innerHTML, } = props;
```
Comment (u12206050) [inline dist/blocks/common/audio.js]: Remove comma

File: dist/blocks/common/cover.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ const WPGCoverBlock = (props) => {
     const { 
     // attrs,
     // innerBlocks,
-    innerHTML } = props;
+    innerHTML, } = props;
```
Comment (u12206050) [inline dist/blocks/common/cover.js]: Remove comma

File: dist/blocks/common/file.js [hunk 3]
```diff
@@ -4,7 +4,7 @@ const WPGFileBlock = (props) => {
     const { 
     // attrs,
     // innerBlocks,
-    innerHTML } = props;
+    innerHTML, } = props;
```
Comment (u12206050) [inline dist/blocks/common/file.js]: Remove comma

File: README.md [hunk 4]
```diff
@@ -98,25 +105,23 @@ import WPGBlocks from 'react-gutenberg'
 
 ### Gatsby Example
 
-```
-import * as React from "react"
-import { graphql } from "gatsby"
-import WPGBlocks from "react-gutenberg"
-import { IWPGBlock } from "react-gutenberg/src/types"
+```tsx
+import * as React from "react";
```
Comment (u12206050) [inline README.md]: Remove ;

**Review by u12206050 — CHANGES_REQUESTED** [review]
Thanks for the effort. Could you remove the `;` at the end of the lines as it isn't really required in JS. Also remove the extra `,` commas that have been added. I commented on some of the lines as example.
Other than those two grammer issues the rest looks very good. Thanks.

**Comment by kuasha420** [comment]
@u12206050 oopsie boopsies. Looks like my prettier did it. Should I add a prettierrc file as well so these doesn't happen again? 

Cheers.

**Comment by u12206050** [comment]
Good idea.

**Comment by kuasha420** [comment]
@u12206050 I've pushed new commit. Please review. Thanks.

File: src/blocks/format/html.tsx [hunk 5]
```diff
@@ -2,12 +2,11 @@ import { IWPGBlock } from '../../types'
...
 
 class WPGHtmlBlock extends React.Component<IWPGBlock> {
-
   id = Math.floor(Math.random() * 100000)
 
   componentDidMount() {
-    document.querySelectorAll(`[data-script="${this.id}"]`).forEach(script => {
-      (window as any).eval(script.innerHTML)
+    document.querySelectorAll(`[data-script="${this.id}"]`).forEach((script) => {
+      ;(window as any).eval(script.innerHTML)
```
Comment (u12206050) [inline src/blocks/format/html.tsx:9]: uhm. Whats going on here?

File: src/blocks/index.tsx [hunk 6]
```diff
@@ -8,28 +8,44 @@ const DefaultHtmlBlock = loadable(() => import('./format/html'))
 
 export function GetTheBlock(name: string) {
   switch (name) {
-    case 'core/audio': return CommonBlocks.Audio
```
Comment (u12206050) [inline src/blocks/index.tsx:11]: I think I much rather prefer the single one line instead of two lines. Makes it easier on the eyes to scan and read

**Review by u12206050 — CHANGES_REQUESTED** [review]
Nearly there

File: src/blocks/format/html.tsx (hunk 5 above)
Comment (kuasha420) [reply src/blocks/format/html.tsx:9]: Confused as well. Prettier keeps putting that semicolon back. Also do you prefer always or avoid for [arrow function parenthesis](https://prettier.io/docs/en/options.html#arrow-function-parentheses)?

File: src/blocks/index.tsx (hunk 6 above)
Comment (kuasha420) [reply src/blocks/index.tsx:11]: Same, but I think it's [not possible](https://github.com/prettier/prettier/issues/6123) with prettier. One workaround is to add this file to prettierignore, but then we lose the benefit of prettier in this file. I'll do as you wish. 

Thanks.

File: src/blocks/format/html.tsx (hunk 5 above)
Comment (u12206050) [reply src/blocks/format/html.tsx:9]: it can be avoid on function parenthesis.

File: src/blocks/index.tsx (hunk 6 above)
Comment (u12206050) [reply src/blocks/index.tsx:11]: Ok no probem. Leave as is. I'll merge now but would be nice if the semicolon issue could be fixed

Comment (kuasha420) [reply src/blocks/index.tsx:11]: I'll try to fix that in my next pr where I fixed a warning about missing keys on column layouts. Thanks for the merge. Cheers!

File: src/blocks/format/html.tsx (hunk 5 above)
Comment (kuasha420) [reply src/blocks/format/html.tsx:9]: Okay I'll configure it to avoid on my next PR.

**Pull request merged**